web/themes/custom/votegov/templates/component/info-card.html.twig

```diff
@@ -0,0 +1,19 @@
+<div class="vote-info-card">
+	<div class="vote-info-card__container">
+		<h3
+			class="vote-info-card__header">{# Field heading content from cms #}
+		</h3>
+		<div class="vote-info-card__body">
+			<p>
+				{# Field body content from cms #}
+			</p>
+		</div>
+		{% if link_href %}
+			<div class="vote-info-card__button">
+				{% include '@votegov/component/button.html.twig' with {
+          {# Field button content from cms #}
+          } %}
+			</div>
+		{% endif %}
```

Comment:
`{% if link %}`

Print out `{{ link.title}}` and `{{ link.url }}`

Reply:
updated link section